compositor_pipeline/src/pipeline/input/rtp.rs

```diff
@@ -158,6 +159,27 @@ fn run_depayloader_thread(
     video_sender: Option<Sender<PipelineEvent<EncodedChunk>>>,
     audio_sender: Option<Sender<PipelineEvent<EncodedChunk>>>,
 ) {
+    let mut audio_eos_received = audio_sender.as_ref().map(|_| false);
+    let mut video_eos_received = video_sender.as_ref().map(|_| false);
```

Comment:
I think this could be simplified by using `bool` instead of `Option<bool>`

Reply:
Yes, but it would only work because we want to do the same action for `None` and `Some(true)`, which are very different states. I prefer to keep the distinction.